src/modules/todolist.js

```diff
@@ -3,15 +3,16 @@ import addToListEnter from './add-tolist-enter';
 import addToList from './add-tolist';
 import removeFromList from './remove-fromlist';
 import toggleDone from './toggle-done';
+import renderDay from './render-day';
 
 window.addEventListener('load', () => {
   const userList = document.querySelector('.user-list');
   const addBtn = document.getElementById('add-btn');
   const list = JSON.parse(localStorage.getItem('list')) || [];
   const addTask = document.getElementById('new-task');
 
+  renderDay(userList);
   renderList(list, userList);
```

Comment:
From such naming is hard to understand what is difference beetween `list` and `userList` , it seems that it should be the same. Let's  give these arguments more clear names, for ex., `listData` and `listContainer` or smth like this